### PR TITLE
Add parachain staking panel

### PR DESCRIPTION
**Pull request by joelamouche**
- Add Staking Panel for parachains (parachainStaking pallet), inpired by the relay chain panel for staking
- Summary view:
  - active collators
  - waiting
  - active / nominators
  - inflation
  - % of inflation for parachain rent
  - % collator commission
  - round view
- Collator list (all candidates)
  - address
  - % collator commission
  - total nominator stake
  - nmbr of nominators
  - own stake
  - min contribution

For Moonriver: 
Screenshot 2021-11-25 at 16 08 17

### CODE REVIEW

**Comment by joelamouche** [comment]
@jacogr ready for review

File: packages/page-staking/src/ParachainStakingApp/index.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  const totalSelected = Number(useCall<string>(api.query.parachainStaking.totalSelected));
+  const totalSelectedStaked = (useCall<BN>(api.query.parachainStaking.staked, [roundInfo?.current]));
+  const inflation = (useCall<{annual: {ideal: ApiResult}}|undefined>(api.query.parachainStaking.inflationConfig));
+  const inflationPrct = inflation?.annual.ideal.toHuman();
+  const parachainBondInfo = (useCall<{percent: ApiResult}|undefined>(api.query.parachainStaking.parachainBondInfo));
+  const parachainBondInfoPrct = parachainBondInfo?.percent.toHuman();
+  const bestNumberFinalized = useBestNumber();
+  const collatorCommission = (useCall<ApiResult|undefined>(api.query.parachainStaking.collatorCommission));
+  // Fetch all collator states using entries
+  const allCollators = useCall<[Uint8Array, CollatorStateRaw][]>((api.query.parachainStaking.candidateState).entries, []);
```
Comment (joelamouche) [inline packages/page-staking/src/ParachainStakingApp/index.tsx:35]: @jacogr how do I make sure this information stays up to date without refreshing the page?

**Comment by fxgamundi** [comment]
Please close in favor of #7275

**Review by rajdeep7Singh — APPROVED** [review]

**Comment by purestaketdb** [comment]
Please do not merge this!

**Comment by jacogr** [comment]
Closed in favor of https://github.com/polkadot-js/apps/pull/7275

**Pull request closed without merging**